.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  target-branch: master
+  reviewers:
+  - MarkEWaite
+  labels:
+  - skip-changelog
```

I think that `dependencies` is a better label in this case, since it will allow the update to appear in the changelog under a section that is dedicated to dependency updates.
```suggestion
  labels:
  - dependencies
```